content/4_Gold/DSU.mdx

```diff
@@ -41,6 +41,15 @@ export const problems = {
 			['DSU'],
 			'usaco-992'
 		),
+		new Problem(
+			'Silver',
+			'Dance Mooves',
+			'1086',
+			'Easy',
+			false,
+			['DSU'],
+			'usaco-1086'
+		),
```

Comment:
keeping this in silver is fine i think more of functional graph than dsu

Reply:
some other silver problems are there w/ dsu and the official solution for silver does a sufficiently good enough expl imo

Reply:
for func graphs that is

Reply:
@dolphingarlic opinion? I think this is a good opportunity to see how DSU can be applied for more silver problems like func graphs